src/app/interactions/query-builder/query-builder-sample-1/query-builder-sample-1.component.ts

```diff
@@ -1,58 +1,92 @@
 import { Component, OnInit } from '@angular/core';
-import { FilteringExpressionsTree, FilteringLogic, IExpressionTree, IgxQueryBuilderComponent, IgxStringFilteringOperand } from 'igniteui-angular';
+import { FilteringExpressionsTree, FilteringLogic, IExpressionTree, IgxBooleanFilteringOperand, IgxDateFilteringOperand, IgxNumberFilteringOperand, IgxQueryBuilderComponent, IgxStringFilteringOperand } from 'igniteui-angular';
 
 @Component({
     selector: 'app-query-builder-sample-1',
     styleUrls: ['./query-builder-sample-1.component.scss'],
     templateUrl: 'query-builder-sample-1.component.html'
 })
 export class QueryBuilderSample1Component implements OnInit {
+    public entities: Array<any>;
+    public companiesFields: Array<any>;
+    public ordersFields: Array<any>;
     public expressionTree: IExpressionTree;
-    
-    public fields: any[] = [
-        { field: 'ID', dataType: 'string' },
-        { field: 'CompanyName', dataType: 'string' },
-        { field: 'ContactName', dataType: 'string' },
-        { field: 'Employees', dataType: 'number' },
-        { field: 'ContactTitle', dataType: 'string' },
-        { field: 'DateCreated', dataType: 'date' },
-        { field: 'TimeCreated', dataType: 'time' },
-        { field: 'Address', dataType: 'string' },
-        { field: 'City', dataType: 'string' },
-        { field: 'Region', dataType: 'string' },
-        { field: 'PostalCode', dataType: 'string' },
-        { field: 'Phone', dataType: 'string' },
-        { field: 'Fax', dataType: 'string' },
-        { field: 'Contract', dataType: 'boolean' }
-    ];
 
     public ngOnInit(): void {
-        const tree = new FilteringExpressionsTree(FilteringLogic.And);
-        tree.filteringOperands.push({
-            fieldName: 'ID',
-            condition: IgxStringFilteringOperand.instance().condition('contains'),
-            searchVal: 'a',
-            ignoreCase: true
+        this.companiesFields = [
+            { field: "ID", dataType: "string" },
+            { field: "CompanyName", dataType: "string" },
+            { field: "ContactName", dataType: "string" },
+            { field: "Employees", dataType: "number" },
+            { field: "ContactTitle", dataType: "string" },
+            { field: "DateCreated", dataType: "date" },
+            { field: "TimeCreated", dataType: "time" },
+            { field: "Address", dataType: "string" },
+            { field: "City", dataType: "string" },
+            { field: "Region", dataType: "string" },
+            { field: "PostalCode", dataType: "string" },
+            { field: "Phone", dataType: "string" },
+            { field: "Fax", dataType: "string" },
+            { field: "Contract", dataType: "boolean" },
```

Comment:
The last comma should be removed
![image](https://github.com/user-attachments/assets/e2d56147-a4a0-4d2d-91b3-c4ce62316975)